hotdoc/utils/utils.py

```diff
@@ -143,39 +149,55 @@ def get_mtime(filename):
         return -1
 
 
+def __load_entry_point(
+        entry_point: meta.EntryPoint, is_installed: bool = False
+        ) -> T.List[T.Type['Extension']]:
+    try:
+        ep_name = entry_point.name
+        activation_function = entry_point.load()
+        classes = activation_function()
+    # pylint: disable=broad-except
+    except Exception as exc:
+        if is_installed:
+            info(f"Failed to load {entry_point}", exc)
+        else:
+            warn('extension-import', f"Failed to load {entry_point} {exc}")
```

Comment:
Oh, I did sneak in a couple of f-strings, which implicitly make hotdoc compatible with Python 3.6+ only.

All earlier Python versions are past end-of-life and no longer supported (heck even Python 3.6 is EOL'd), but I did notice there's still a Python 3.5 check in `setup.py`. If compatibility with older versions is intended, I can take those back out.

Reply:
Never mind, there were already f-strings in the file.